packages/expect/src/matchers.js

```diff
@@ -628,27 +628,20 @@ const matchers: MatchersObject = {
       true,
     );
 
+    const hint = matcherHint('.toStrictEqual', undefined, undefined, {
+      isNot: this.isNot,
+    });
     const message = pass
       ? () =>
-          matcherHint('.not.toStrictEqual') +
+          hint +
           '\n\n' +
-          `Expected value to not equal:\n` +
-          `  ${printExpected(expected)}\n` +
-          `Received:\n` +
-          `  ${printReceived(received)}`
+          `Expected: ${printExpected(expected)}\n` +
+          `Received: ${printReceived(received)}\n`
```

Comment:
I've updated this in the code. I just did a `commit --amend --no-edit` and forced push. Hence, you're not gonna see a new commit. Just in case you get confused by the lack of commits. :)